src/js/drawer.js

```diff
@@ -116,7 +116,15 @@ class Drawer {
 			headerTop.appendChild(this.relatedContent);
 		}
 
-		this.nav.classList.toggle(this.class.drawer, this.enabled);
+		// Toggle drawer classlist. Cannot use `toggle` as IE11 does not support
+		// the `toggle` method, and adding a new polyfill requirement is a
+		// breaking change.
+		if(this.enabled) {
+			this.nav.classList.add(this.class.drawer);
+		}
+		if(!this.enabled) {
+			this.nav.classList.remove(this.class.drawer);
+		}
```

Comment:
I don't think it would be a breaking change in this instance because the toggle method has been used in the component already for some amount of time.

Reply:
Yeah that's a good point but I opted to treat this as a bug incase its been broken for IE11 users in projects this whole time without anyone noticing 😬 E.g. if the project isn't including the optional `ResizeObserver` polyfill an error would have been thrown and the core experience would have worked reasonably well.